package.json

```
@@ -27,10 +27,17 @@
 		"test:coverage:report": "vitest run --coverage --reporter=html",
 		"start": "node dist/main.js"
 	},
-	"keywords": [],
-	"author": "",
+	"keywords": [
```

```suggestion
	"keywords": [
	      "internet computer",
          "internet-computer",
          "Internet Computer",
          "ic",
          "dfinity",
```
I compared with other Dfinity packages, e.g. https://github.com/dfinity/ic-js/blob/fa428e004ed3f8dde3466e2c0c9a150ab68e9134/packages/ckbtc/package.json#L42